docs/supplychain-sbom.md

```diff
@@ -115,17 +115,22 @@ The color-coding is used in this document to help illustrate different SBOM acti
 > [!NOTE]
 > FIXME – Check if this is sane.
 
+<!--
+Note from Josh
+I'm not entirely convinced an assembler should exist as a section. It might make more sense to just refer to this as an author then break it out someday if that makes more sense.
```

Comment:
Would you mind expanding on your thoughts on the "SBOM Assembler" role?

My reason for proposing a role that is separate from "SBOM Author" is the same as distinguishing between "Create" and "Update" in CRUD – to clarify responsibilities and expectations for were fields come from – assuming some fields may be updated as an SBOM is passed down a supply-chain (e.g. to correct upstream assumptions like 'download location' or whatever).

Have I made a bad assumption here?

Reply:
I'm unsure this role will ever exist. The author will already do almost all of this. And if someone does make SBOM changes, wouldn't they just become the new author? This feels a bit like forking an open source project

Reply:
Allright, I've given this a think, and I think I've landed on your side of the matter...

My reasoning is that "SBOM Authors" aren't about "what" SBOM metadata is being edited or created, but rather "if" metadata in an SBOM is being edited or created.

The "what" is decided by the Supply Chain Role alone.

I'll update the texts accordingly; You can safely remove your comment in the file. :-)

Reply:
One thing, though.

I still think it's useful to clearly distinguish between authoritative sources of metadata from those downstream. 

One thing though. I've added a bunch of 🟨 boxes all over the place, that don't have a clear definition when we merge the "🟨 SBOM Assembler" role into the "🟥 SBOM Author" role. 

How would you communicate the distinction between the situation when someone is the upstream source of  a metadata field, and anyone downstream that may "update" or "fix" a field?

Reply:
I _think_ I've found a... decent? way to get my point across.

Good if you can take a look at what I pushed to the supplychain-sbom branch. :-)

Reply:
Yeah, I like that!

Reply:
Allright, I'll resolve this  after you've removed the inline comment :-)